features/manage_response_sets.feature

```diff
@@ -72,11 +72,11 @@ Feature: Manage Response Sets
     Then I should see "Changes by test_author@gmail.com"
     And I should see "field changed from"
     And I should not see "No changes have been made to this version."
-    When I click on the "Update Tags" link
-    And I fill in the "Tags" field with "TagTest2"
-    And I tab out of the "Tags" field
-    And I click on the "Save" button
-    Then I should see "TagTest1, TagTest2"
```

Comment:
Is this removed because the tags are on a new tab or?

Reply:
The "Update Tags" link button no longer has text, only an image.  I couldn't find an example and was going to do a little more research and put it back in after the next re-work.

Reply:
pulled dev, rebased and re-pushed.